Simplenote/SPAppDelegate.m

```diff
@@ -189,12 +190,19 @@ - (void)applicationDidEnterBackground:(UIApplication *)application
     [self cleanupScrollPositionCache];
     [self syncWidgetDefaults];
     [self resetWidgetTimelines];
+
+    self.selectedNotesEnteringBackground = self.noteListViewController.tableView.indexPathsForSelectedRows;
```

Comment:
Same note here, perhaps we could move this into the Note List itself?

(The appDelegate could relay the event, or the Note List could listen to the `didEnterBackground` notification)

Reply:
done